bitcoin/BRPeerManager.c

```diff
@@ -839,7 +839,7 @@ static void _peerDisconnected(void *info, int error)
         if (manager->connectFailureCount > MAX_CONNECT_FAILURES) manager->connectFailureCount = MAX_CONNECT_FAILURES;
     }
 
-    if (! manager->isConnected && manager->connectFailureCount == MAX_CONNECT_FAILURES) {
+    if (! manager->isConnected && manager->connectFailureCount >= MAX_CONNECT_FAILURES) {
```

Comment:
I'd seen cases where this was greater than MAX_CONNECT_FAILURES. In `BRPeerManagerConnect` and `BRPeerManagerPublishTx`, we checked for `>=`, mirrored that here as I had seen cases where it was greater than at this point.